docs/CreateAKSClusterForScalarDB.md

```diff
@@ -0,0 +1,156 @@
+* [Azure CLI](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli)
+* [PowerShell](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
+* [Azure portal](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-portal)
+
+## Requirements
+
+When deploying ScalarDB Server, you must:
+
+* Create the AKS cluster by using Kubernetes version 1.21 or higher.
+* Configure the AKS cluster based on the version of Kubernetes and your project's requirements.
+
+## Recommendations (optional)
+
+The following are some recommendations for deploying ScalarDB Server. These recommendations are not required, so you can choose whether or not to apply these recommendations based on your needs.
+
+### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDB Server node pool
+
+From the perspective of commercial licenses, resources for one pod running ScalarDB Server are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDB Server pod and one Envoy pod on one worker node.
+
+In other words, the following components run on one worker node:
+
+* ScalarDB Server pod (2vCPU / 4GB)
+* Envoy proxy (0.2–0.3 vCPU / 256–328 MB)
+* Kubernetes components
+
+With this in mind, you should use a worker node that has 4vCPU / 8GB memory resources. We recommend running only the above components on the worker node for ScalarDB Server. However, if you want to run other pods on the worker node for ScalarDB Server, you should use a worker node that has more than 4vCPU / 8GB memory.
+
+Note that you should configure resource limits based on your system's workload if the Envoy pod exceeds the above resource usage. In addition, you should consider scaling out the worker node and the ScalarDB Server pod if the ScalarDB Server pod exceeds the above resource usage and if latency is high (throughput is low) in your system.
+
+### Create a node pool for ScalarDB Server pods
+
+AKS creates one system node pool named **agentpool** that is preferred for system pods (used to keep AKS running) by default. We recommend creating another node pool with **user** mode for ScalarDB Server pods and deploying ScalarDB Server pods on this additional node pool.
+
+### Create a node pool for other, non-ScalarDB Server application pods
+
+We recommend running only ScalarDB Server pods on the worker node (node pool) for ScalarDB Server. If you want to run other application pods on the same AKS cluster, you should create other node pools for your application pods.
+
+### Create a node pool for monitoring components (kube-prometheus-stack and loki-stack)
+
+We recommend running only ScalarDB Server pods on the worker node (node pool) for ScalarDB Server. If you want to run monitoring pods (e.g., Prometheus, Grafana, Loki, etc.) by using [kube-prometheus-stack](./K8sMonitorGuide.md) and [loki-stack](./K8sLogCollectionGuide.md) on the same AKS cluster, you should create other node groups for monitoring pods.
```

Comment:
```suggestion
We recommend running only ScalarDB Server pods on the worker node (node pool) for ScalarDB Server. If you want to run monitoring pods (e.g., Prometheus, Grafana, Loki, etc.) by using [kube-prometheus-stack](./K8sMonitorGuide.md) and [loki-stack](./K8sLogCollectionGuide.md) on the same AKS cluster, you should create other node pools for monitoring pods.
```

Reply:
Ah, sorry. I overlooked this point in my previous updates.
Thank you for pointing it out!